gen/calc.py

```diff
@@ -222,6 +222,20 @@ def validate_oauth_enabled(oauth_enabled):
     validate_true_false(oauth_enabled)
 
 
+def validate_network_default_name(dcos_overlay_network_default_name, dcos_overlay_network):
+    try:
+        overlay_network = json.loads(dcos_overlay_network)
+    except ValueError:
+        # TODO(cmaloney): This is not the right form to do this
```

Comment:
This comment tells me we shouldn't be copy/pasting this form.
I think it should be:
```
try:
   foo()
except ValueErroras ex:
    raise AssertionError(my message) from ex
```
@cmaloney Is that what you were aiming at with this comment?